File: codewars/8kyu/area-or-perimeter/dduck-area-or-perimeter.js
// https://www.codewars.com/kata/5ab6538b379d20ad880000ab

const areaOrPerimeter = (l, w) => {
  if (l === w) {
    return l ** 2;
  }
  return (l + w) * 2;
};

console.log(areaOrPerimeter(4, 4), 16);
console.log(areaOrPerimeter(6, 10), 32);

File: codewars/8kyu/area-or-perimeter/dduck-area-or-perimeter.py
# https://www.codewars.com/kata/5ab6538b379d20ad880000ab

from unittest import TestCase


def area_or_perimeter(l, w):
    if l == w:
        return l ** 2
    return (l + w) * 2


TestCase().assertEqual(area_or_perimeter(4, 4), 16)
TestCase().assertEqual(area_or_perimeter(6, 10), 32)
